CHANGELOG.md

```diff
@@ -19,11 +19,22 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
     -   editor is focused on load if `autoFocus` prop is set to `true`
     -   implemented support for `disabled` state in code editor
     -   implemented support for `intent` states in code editor
+    -   added custom toolbar for markdown mode. Had to change the typescript target to `es2018` to accommodate `codemirror-toolbar` package.
```

Comment:
This states something different from the current change in `.typescript/tsbuild-cjs.json`.